lldb/source/Core/CoreProperties.td

```diff
@@ -5,10 +5,11 @@ let Definition = "modulelist" in {
     Global,
     DefaultTrue,
     Desc<"Control the use of external tools and repositories to locate symbol files. Directories listed in target.debug-file-search-paths and directory of the executable are always checked first for separate debug info files. Then depending on this setting: On macOS, Spotlight would be also used to locate a matching .dSYM bundle based on the UUID of the executable. On NetBSD, directory /usr/libdata/debug would be also searched. On platforms other than NetBSD directory /usr/lib/debug would be also searched. If all other methods fail there may be symbol-locator plugins that, if configured properly, will also attempt to acquire symbols. The debuginfod plugin defaults to the DEGUFINFOD_URLS environment variable which is configurable through the 'plugin.symbol-locator.debuginfod.server_urls' setting.">;
-  def EnableBackgroundLookup: Property<"enable-background-lookup", "Boolean">,
```

Comment:
Should we leave this setting around and adapt it to use the new setting with the correct defaults set? If someone has a `settings set enable-background-lookup true` in their ~/.lldbinit file, this will cause the .lldbinit file to stop executing lines if this setting fails to be set. This is kind of like a backward compatibility in our API kind of thing. We can make this setting as deprecated and tell people to use the `download` setting instead, but it would be nice if it still worked.

If this was used for dsymForUUID only and there weren't many customers it could be ok to remove, but I would err on the side of caution.

Reply:
I was on the fence, as I've been encouraging users to use  `settings set symbols.enable-background-lookup true` in their `.lldbinit` (because older versions don't know about the setting either). On the other hand it's very little work to keep the old setting working, so why not. I've pushed a commit that makes the old setting an alias.